schema/method_units/method_unit_base.json

```diff
@@ -0,0 +1,41 @@
+{
+    "schemaId": "method-unit-base",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "unit method schema (base)",
+    "properties": {
+        "type": {
+            "description": "General type of the method",
+            "$ref": "../core/primitive/slugified_entry_or_slug.json"
+        },
+        "subtype": {
+            "description": "General subtype of the method (if applicable)",
+            "$ref": "../core/primitive/slugified_entry_or_slug.json"
+        },
+        "parameters": {
+            "description": "Keyworded collection of method parameters (numerically relevant)",
+            "type": "object"
+        },
+        "precision": {
+            "description": "Object showing the actual possible precision based on theory and implementation",
+            "type": "object"
+        },
+        "data": {
+            "description": "Additional numerically relevant data specific to method, eg. array of pseudopotentials",
+            "type": "object"
+        },
+        "metaData": {
```

Comment:
Should we just use all lowercase - `metadata` for simplicity?

Reply:
Ok, yes.